### DISCUSSION TITLE
Unknown directive "@_unmask".

### DISCUSSION THREAD

**Discussion by WonderPanda** [0no-co/gql.tada#55] [Q&A] [answered]
This appears to be a false positive as applying the directive definitely seems to unmask the fragment as expected.

Its distracting seeing red errors in the IDE will writing graphql operations though. Is this a known issue that anyone has faced?

![image](https://github.com/0no-co/gql.tada/assets/3631771/5f40d89d-b6e1-470c-adc6-605e8c19465a)

**Comment by JoviDeCroock** [comment, marked as answer]
Have you tried updating your version of `GraphQLSP`, this has been fixed there for a while

**Reply by WonderPanda** [reply]
I'm on the latest version of GraphQLSP. 

This error seems to pop up periodically. I did go through and disable a couple VsCode extensions related to GraphQL and the problem seems to be gone now.

**Reply by WonderPanda** [reply]
It seems to be back again periodically. Killing TSServer and rebooting it will sometimes make it go away and sometimes not.

Our schema is pretty large and I'm definitely seeing some performance penalties for type checking. Just ran into our first "type instantiation is excessively deep and possibly infinite" trying to move some of our more complex queries over

**Reply by WonderPanda** [reply]
@JoviDeCroock We've successfully set up our application to start using gql.tada! Its an incredible developer experience. We're still seeing this error periodically show up and disappear across multiple dev machines and between VsCode and NeoVim though. Let me know if there's any kind of diagnostic info I can help pull that could potentially help identify the issue :pray:

**Reply by Danbka** [reply]
@WonderPanda I have the same issue Did you solve it?

**Reply by WonderPanda** [reply]
No, we still get transient error reports all over the place with @_unmask. We're just ignoring it right now because it doesn't impact any functionality but it would definitely be awesome if this could be fixed
